requirements.txt

```diff
@@ -8,12 +8,12 @@ Flask-Login==0.4.1
 Flask-Mail==0.9.1
 Flask-Migrate==2.1.1
 Flask-SQLAlchemy==2.3.2
-Flask-WTF==0.14.2
+Flask-WTF==0.14.3
 Pillow>=5.2.0
 psycopg2>=2.6.2 --no-binary psycopg2
 SQLAlchemy==1.1.15
 gunicorn==19.9
 Fabric3==1.14.post1
 itsdangerous==0.24
 us>=1.0
-future
```

Comment:
There wasn't a newline at the end of the file; my editor automatically added it. I decided to leave the change in since many editors auto-add the newline, and someone will eventually run into it.